learning_paths/api/v1/views.py

```diff
@@ -23,3 +25,19 @@ class LearningPathAsProgramViewSet(viewsets.ReadOnlyModelViewSet):
     permission_classes = (IsAuthenticated,)
     serializer_class = LearningPathAsProgramSerializer
     pagination_class = PageNumberPagination
+
+
+def learning_path_progress_view(request, learning_path_id):
```

Comment:
Nit: shouldn't we use the DRF `APIView` and a serializer to keep it consistent with the other view? Also, if we plan to add the view to list multiple users' progress in a learning path, it could make sense to call this something like `LearningPathUserProgressView`.

We should also remember to use `permission_classes = (IsAuthenticated,)`. Currently, this endpoint raises the `AttributeError` when accessed by an anonymous user.

Reply:
Updated it to use APIView with the permission class.